contracts/JBETHPaymentTerminal.sol

```diff
@@ -694,8 +721,19 @@ contract JBETHPaymentTerminal is
     address _beneficiary,
     string memory _memo
   ) private returns (uint256 feeAmount) {
+    // Get the fee discount.
+    uint256 _feeDiscount = feeGuage == IJBFeeGuage(address(0))
+      ? 0
+      : feeGuage.currentDiscountFor(_projectId);
+
+    // Set the discounted fee if its valid.
+    if (_feeDiscount > 1000000) _feeDiscount = 0;
```

Comment:
need to define the const for MAX_FEE_DISCOUNT

Reply:
@mejango why not` > 1000000) _feeDiscount = 1000000;` ?

Reply:
yeah that works also. 

Make sure to pull this out into a const in the lib too plz!!

Reply:
also @drgorillamd might as well pump up the fidelity of the const. I think i just plopped a random number when i was doing it.

Reply:
done, same granularity (10**9) - I kepr the '=0', if there is a gauge bug, maybe easier to give back extra fee perceived than to ask for such ':)